cmd/site-controller/controller.go

```diff
@@ -209,13 +220,21 @@ func (c *SiteController) checkSite(key string) error {
 		log.Println("Error checking skupper-site namespace: ", err)
 		return err
 	}
-	//get skupper-site configmap
+	// get skupper-site configmap
 	obj, exists, err := c.siteInformer.GetStore().GetByKey(key)
 	if err != nil {
 		log.Println("Error checking skupper-site config map: ", err)
 		return err
 	} else if exists {
 		configmap := obj.(*corev1.ConfigMap)
+		// if policy is enabled, ensure ClusterRoleBinding exists even for not controlled sites
+		if c.validator.Enabled() {
+			c.addClusterRoleBinding(configmap)
+		}
+		// If site is not controlled, ignore it
+		if !c.isSiteControlled(configmap.ObjectMeta.Namespace) {
```

Comment:
Why not just check configmap above rather than re-retrieving it from the api server?